### PR TITLE
Fix usages of switch statement

### PR DESCRIPTION
**Pull request by davido**

### CODE REVIEW

File: src/com/facebook/buck/apple/project_generator/ProjectGenerator.java [hunk 1]
```diff
@@ -2716,17 +2716,14 @@ ProductType bundleToTargetProductType(
           if (productType.isPresent()) {
             return productType.get();
           }
-        } else {
-          switch (extension) {
-            case FRAMEWORK:
-              return ProductType.STATIC_FRAMEWORK;
-          }
+        } else if (extension == FRAMEWORK) {
```
Comment (cushon) [inline src/com/facebook/buck/apple/project_generator/ProjectGenerator.java]: This makes the `@SuppressWarnings("incomplete-switch")` of the method unnecessary, and `FRAMEWORK` needs to be be `AppleBundleExtension.FRAMEWORK`.

File: src/com/facebook/buck/apple/project_generator/ProjectGenerator.java [hunk 2]
```diff
@@ -2716,17 +2716,14 @@ ProductType bundleToTargetProductType(
...
+            return ProductType.STATIC_FRAMEWORK;
+	  }
         }
       } else if (
           binaryNode.getDescription() instanceof AppleBinaryDescription) {
-        switch (extension) {
-          case APP:
-            return ProductType.APPLICATION;
+        if (extension == APP) {
```
Comment (cushon) [inline src/com/facebook/buck/apple/project_generator/ProjectGenerator.java]: `AppleBundleExtension.APP`

File: third-party/java/aosp/src/com/android/manifmerger/MergingReport.java [hunk 3]
```diff
@@ -322,6 +322,8 @@ Builder addMessage(@NonNull SourceFilePosition sourceFilePosition,
                 case WARNING:
                     mHasWarnings = true;
                     break;
+		default:
```
Comment (cushon) [inline third-party/java/aosp/src/com/android/manifmerger/MergingReport.java]: Some of this indentation is off.

File: third-party/java/aosp/src/com/android/manifmerger/PostValidator.java [hunk 4]
```diff
@@ -200,6 +200,8 @@ private static void validate(
                             ));
                 }
                 break;
+	    default:
+		// fall out
```
Comment (cushon) [inline third-party/java/aosp/src/com/android/manifmerger/PostValidator.java]: Indentation

File: src/com/facebook/buck/apple/project_generator/ProjectGenerator.java [hunk 5]
```diff
@@ -2716,17 +2716,14 @@ ProductType bundleToTargetProductType(
...
           }
-        } else {
-          switch (extension) {
-            case FRAMEWORK:
-              return ProductType.STATIC_FRAMEWORK;
-          }
+        } else if (extension == FRAMEWORK) {
+            return ProductType.STATIC_FRAMEWORK;
+	  }
         }
```
Comment (davido) [inline src/com/facebook/buck/apple/project_generator/ProjectGenerator.java]: Done

File: src/com/facebook/buck/apple/project_generator/ProjectGenerator.java (hunk 2 above)
Comment (davido) [inline src/com/facebook/buck/apple/project_generator/ProjectGenerator.java]: Done

File: third-party/java/aosp/src/com/android/manifmerger/MergingReport.java (hunk 3 above)
Comment (davido) [inline third-party/java/aosp/src/com/android/manifmerger/MergingReport.java]: Ah, tabs. M-x untabify.

Thanks, Done.

File: third-party/java/aosp/src/com/android/manifmerger/PostValidator.java (hunk 4 above)
Comment (davido) [inline third-party/java/aosp/src/com/android/manifmerger/PostValidator.java]: Same here, tabs, Done.

**Comment by davido** [comment]
@marcinkosiba Fixed some issues. PTAL.

**Pull request closed without merging**